sphericart/include/cuda_base.hpp

```diff
@@ -66,29 +53,29 @@ namespace cuda {
  */
 template <typename scalar_t>
 void spherical_harmonics_cuda_base(
-    const scalar_t* __restrict__ xyz,
+    const scalar_t* xyz,
     const int nedges,
-    const scalar_t* __restrict__ prefactors,
+    const scalar_t* prefactors,
     const int nprefactors,
     const int64_t l_max,
     const bool normalize,
     const int64_t GRID_DIM_X,
     const int64_t GRID_DIM_Y,
     const bool gradients,
     const bool hessian,
-    scalar_t* __restrict__ sph,
-    scalar_t* __restrict__ dsph,
-    scalar_t* __restrict__ ddsph,
+    scalar_t* sph,
+    scalar_t* dsph,
+    scalar_t* ddsph,
     void* cuda_stream = nullptr
 );
 
 template <typename scalar_t>
 void spherical_harmonics_backward_cuda_base(
-    const scalar_t* __restrict__ dsph,
-    const scalar_t* __restrict__ sph_grad,
+    scalar_t* dsph,
+    scalar_t* sph_grad,
     const int nedges,
     const int ntotal,
-    scalar_t* __restrict__ xyz_grad,
+    scalar_t* xyz_grad,
```

Comment:
I see that the consts were added back to the forward equivalent of this. Would it be possible to do the same here? (Not high priority, I see that the API is now back to const)

Reply:
this should be resolved now.